packages/git-semver-tags/index.js

```diff
@@ -38,6 +38,12 @@ module.exports = function(callback, opts) {
           if (lernaTag(tag, opts.package)) {
             tags.push(tag);
           }
+        } else if (opts.tagPrefix) {
+          if ((new RegExp('^' + opts.tagPrefix)).test(tag)) {
```

Comment:
Instead of constructing a regular expression on every tag, could this be moved outside of the `forEach` loop?

Also, does `tagPrefix` need to be escaped? It doesn't say anywhere that the `tagPrefix` can be a regular expressions. Therefore maybe it needs to be escaped before being passed to `RegExp`.